src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div
+        class="card-block"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+        />
+        <h3 class="card-block__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card-block__code">Product code: 195434</p>
+        <div class="card-block__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card-block__text">Reviews: 5</p>
+        </div>
+        <div class="card-block__price-block price-block">
+          <p class="price-block__text">Price:</p>
+          <p class="price-block__number">$2,199</p>
+        </div>
+        <a
+          class="button"
```

Comment:
there should be just 2 BEM blocks card and stars button should be an element of card block